test/test_function_data.py

```diff
@@ -88,7 +89,36 @@ def test_extract_by_subinterval(self):
                         self.assertTrue(isinstance(f, Function))
 
 
-class TestForwardSolutionData(BaseTestCases.TestFunctionData):
+class TestSteadyForwardSolutionData(BaseTestCases.TestFunctionData):
+    """
+    Unit tests for :class:`~.ForwardSolutionData`.
+    """
+
+    def setUp(self):
+        end_time = 1.0
+        self.num_subintervals = 1
+        timesteps = [1.0]
+        self.field = "field"
+        self.num_exports = [1]
+        self.mesh = UnitTriangleMesh()
+        self.time_partition = TimePartition(
+            end_time, self.num_subintervals, timesteps, self.field
+        )
+        self.function_spaces = {
+            self.field: [
+                FunctionSpace(self.mesh, "DG", 0) for _ in range(self.num_subintervals)
+            ]
+        }
+        self._create_function_data()
```

Comment:
This is duplicated across the new tests you introduce. Please introduce another base test case.

Reply:
done in commit [1c328d6](https://github.com/pyroteus/goalie/pull/149/commits/1c328d6bc1550e27e7250d7c686ac3871e755180)